lib/ansible/galaxy/role.py

```diff
@@ -260,9 +269,9 @@ def install(self):
                     except:
                         raise AnsibleError("this role does not appear to have a valid meta/main.yml file.")
 
-                # we strip off the top-level directory for all of the files contained within
-                # the tar file here, since the default is 'github_repo-target', and change it
-                # to the specified role's name
+                # we strip off any higher-level directories for all of the files contained within
+                # the tar file here. The default is 'github_repo-target'. Gerrithub, on the other
```

Comment:
Small nit, Gerrit is the actual software. Gerrithub is a hosted Gerrit instance.